resources/js/textlimit.js

```diff
@@ -0,0 +1,23 @@
+$(function() {
```

Comment:
if this is to keep the number of lines whatever screen size you are in, you can just use css ellipsis for this
here's an example on how to do it https://stackoverflow.com/questions/5269713/css-ellipsis-on-second-line